app/src/main/java/org/mozilla/focus/fragment/InfoFragment.java

```diff
@@ -50,10 +57,24 @@ public View inflateLayout(LayoutInflater inflater, @Nullable ViewGroup container
             webView.setVisibility(View.INVISIBLE);
         }
 
+        applyLocale();
```

Comment:
This fragment already extends from LocaleAwareFragment. I wonder if we should move this into LocaleAwareFragment - I guess all fragments would need to do this?